sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -537,6 +537,16 @@ def text(self, encoding=None):
             encoding = "utf-8-sig"
         return self.body().decode(encoding)
 
+    def json(self):
+        # type: () -> Dict[str, Any]
```

Comment:
Let's avoid being more constraints than the stdlib lib itself:
https://github.com/python/typeshed/blob/27f38b8c23da0a2d0ff9ce486f84664c8b9a2d58/stdlib/json/__init__.pyi#L46

Any should do it